resources/localization/de/PrusaSlicer_de.po

```diff
@@ -6998,7 +7172,7 @@ msgstr "Dreiecke"
 
 #: src/libslic3r/PrintConfig.cpp:3059
 msgid "Try to repair any non-manifold meshes (this option is implicitly added whenever we need to slice the model to perform the requested action)."
-msgstr "Repariere alle ungeschlossenen Netze (diese Option wird implizit hinzugefügt, wenn wir das Modell slicen müssen, um die gewünschte Aktion ausführen zu können)."
+msgstr "Repariere alle ungeschloßenen Netze (diese Option wird implizit hinzugefügt, wenn wir das Modell slicen müssen, um die gewünschte Aktion ausführen zu können)."
```

Comment:
"ungeschlossenen" is correct, "ungeschloßenen" is wrong.

Reply:
correct.